Automattic/pocket-casts-android
# Use Play Store referrals info

## Description
This displays offer duration and price on referral screens using hard-coded test product id and offer id.

## Testing Instructions
1. Launch the `debugProd` or `debug` build variant
2. Login with a paid account
3. Go to the profile screen
4. ✅ Notice that referral screens are populated using play store offer info
6. Change currency price or currency code in `yearlyTestPlusProduct` of `ProductDetailsModule` of the installed build variant
7.  Go to the profile screen and tap the referral  banner
8. ✅ Notice that currency price and code are displayed correctly 
9. Remove `yearlyTestPlusProduct` or change its `productId`/ `offerId` in `ProductDetailsModule` of the installed build variant
10. Reinstall the app
11. ✅ Notice that referral icon and the banner are not shown if the product or offer does not exist in product details

## Screenshots or Screencast 
Profile | Send Pass | Claim Pass
---|---|--
<img width=200 src="https://github.com/user-attachments/assets/c495c631-cdba-4328-b86b-db028821304a"/>|<img width=200 src="https://github.com/user-attachments/assets/bede35ea-0fb2-4fdb-adc7-970a9c342d64"/>|<img width=200 src="https://github.com/user-attachments/assets/2ae40700-ecdb-4c4a-aad5-b0ec67e9bb4f"/>


## Checklist
 
#### I have tested any UI changes...
